index.html

```diff
@@ -6,168 +6,153 @@
+        <li class="Skill-list">Frameworks <a href="#"><img class="Pointer-2"  src="images/Union.png" alt="icon"> <li></li></li></a>
+        <img src="images/Separator Bottom.png" alt="icon">
+        <li class="Skill-list">Skills <a href="#"><img src="images/Union.png" class="Pointer-3" alt="icon"> <li></li></li></a>
+        <img src="images/Separator Bottom.png" alt="icon">
       </ul>
-        <a href="#"><button type="button" class="box">
-            <p>See Project</p>
-        </button></a>
       </section>
     </section>
-  </section>
-  </section>
-  <section class="about">
-    <h2>About<br>Myself</h2>
-    <p>
-      Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
-    </p>
-    <h3 id="con">LETS CONNECT</h3>
-    <ul class="socials">
-      <li><a href="#"><img src="images/Tweeter.png"></a></li>
-      <li><a href="#"><img src="images/Linkedin icon.png"></a></li>
-      <li><a href="#"><img src="images/m.png"></a></li>
-      <li><a href="#"><img src="images/Github.png"></a></li>
-      <li><a href="#"><img src="images/tech.png"></a></li>
-    </ul>
-    <a href="#"><button type="button" id="resume" class="box">Get my resume</button></a>
-    <ul class="skills">
-      <li class="skill-card">
-        <h3>Languages</h3>
-        <img id="vi" src="images/Union - Copy.png" alt="arrow">
-        <ul>
-          <li class="skill-li"><img src="images/Ellipse 1.png">Javascript</p></li>
-          <li class="skill-li"><img src="images/html.png">Html</li>
-          <li class="skill-li"><img src="images/css (2).png">Css</li>
-        </ul>
-      </li>
-      <li class="skill-card">
-        <h3> Frameworks</h3>
-        <img id="un" src="images/Union.png">
-        <img src="images/Separator Bottom.png" alt="line">
-      </li>
-        <ul></ul>
-      <li class="skill-card skill-card1">
-        <h3>Skills</h3>
-        <img id="co" src="images/Union.png">
-        <img src="images/Separator Bottom.png" alt="line">
-        <ul></ul>
-      </li>    
-    </ul>
-  </section>      
+    <section class="Contact-me">
+      <h3 id="Me">Contact me</h3>
+      <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+      <form action="https://formspree.io/f/mqkjgawz" method="post" name="Form" class="Click-1">
+        <input type="text" class="Details" name="name" placeholder="Uchechi" maxlength="30" required>
+        <input type="email" class="Details" name="name" placeholder="nwakauc1@gmail.com" required>
+        <textarea name="Text" id="Details" cols="50" rows="10" placeholder="Write your message here" maxlength="500" required></textarea>
+        <button id="Get-in-touch" class="button-2" type="submit" target="_blank" action="submit">Get in touch</button>  
```

Comment:
- You have not applied the right hover styling on your button, please try to fix this. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏. You can consider fixing this for other buttons as they may apply 👍 👍 

<img width="367" alt="hover2" src="https://user-images.githubusercontent.com/73607512/199258176-9218e9ba-e832-4a50-b3b2-3479c10da92e.PNG">